src/__tests__/getJavaOptions.test.ts

```diff
@@ -24,6 +24,15 @@ describe("getJavaOptions", () => {
     expect(options).toEqual(proxyOptions);
   });
 
+  const malformedOptions = ["-XX:+UseNUMA   ", "-XX:+UseZGC  "];
+  const expectedOptions = ["-XX:+UseNUMA", "-XX:+UseZGC"];
```

Comment:
minor, but since these two const are only used by a single test, can you move them inside?

Reply:
Done!